### PR TITLE
Update styling conventions and structure

### PR DESCRIPTION
**Pull request by braxex**
# Update styling conventions and structure

# Summary

Updates styling convention to scope styles tightly to components (avoiding polluting the global namespace). 

Addresses #16 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How To Test

# Checklist:

- [x] My code follows the style guidelines of this project and has no linting errors
- [x] I have performed a self-review of my code
- ~[ ] I have commented my code, particularly in hard-to-understand areas~
- ~[ ] I have made corresponding changes to the documentation, including any applicable ADRs~
- [x] My changes generate no new warnings
- ~[ ] I have added tests that fail without these changes~
- ~[ ] New and existing tests (unit, integration, accessibility) pass locally~
- ~[ ] Documentation updated~
- ~[ ] If there are security concerns they are addressed or ticketed after being discussed~

### CODE REVIEW

File: app/styles/styles.scss [hunk 1]
```diff
@@ -1,7 +1,7 @@
 @forward "uswds-theme.scss";
+@forward "globals.scss";
```
Comment (ntsummers1) [inline app/styles/styles.scss:2]: We'll want to keep this below the `@forward "usa-button"` import. The [USWDS style guide](https://github.com/uswds/uswds?tab=readme-ov-file#sass-and-theme-settings) recommends it comes second and all other custom styles follow it.

File: app/styles/styles.scss [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
 
 // Import USWDS components as needs for optimal build speed
```
Comment (ntsummers1) [inline app/styles/styles.scss:4]:
```suggestion
// Import USWDS components as needed for optimal build speed
```

**Review by ntsummers1 — CHANGES_REQUESTED** [review]

File: app/styles/styles.scss (hunk 1 above)
Comment (braxex) [reply app/styles/styles.scss:2]: 👍 Fixed.

File: app/styles/styles.scss (hunk 2 above)
Comment (braxex) [reply app/styles/styles.scss:4]: 👍 Fixed

**Pull request merged**